combinations/hash.tsv

```diff
@@ -13,8 +13,8 @@ bwa-mem2=2.2.1,samtools=1.12	bgruening/busybox-bash:0.1	0
 bwa-mem2=2.2.1,samtools=1.14	bgruening/busybox-bash:0.1	0
 bwa-mem2=2.2.1,samtools=1.15	bgruening/busybox-bash:0.1	0
 bwa-mem2=2.2.1,samtools=1.15.1	bgruening/busybox-bash:0.1	0
-hisat2=2.2.0,samtools=1.10	bgruening/busybox-bash:0.1	0
-hisat2=2.2.0,samtools=1.15.1	bgruening/busybox-bash:0.1	0
+hisat2=2.2.1,samtools=1.10	bgruening/busybox-bash:0.1	0
```

Comment:
```suggestion
```

Reply:
We should be fine with just a container for the latest version of `samtools` right @RaqManzano ?

Reply:
Absolutely!